assets/js/app.js

```diff
@@ -12,5 +12,20 @@ import "../css/app.css";
 //     import {Socket} from "phoenix"
 //     import socket from "./socket"
 //
-import "phoenix_html"
+
 import 'alpinejs'
+import "phoenix_html"
+import {Socket} from "phoenix"
+import {LiveSocket} from "phoenix_live_view"
+
+let csrfToken = document.querySelector("meta[name='csrf-token']").getAttribute("content")
+let liveSocket = new LiveSocket("/live", Socket, {params: {_csrf_token: csrfToken}})
```

Comment:
Based on this [blog entry](https://dockyard.com/blog/2020/12/21/optimizing-user-experience-with-liveview) I think we need a small blurb for Alpine here:

```
import Alpine from "alpinejs"
let liveSocket = new LiveSocket("/live", Socket, {
  params: {_csrf_token: csrfToken},
 dom: {
   onBeforeElUpdated(from, to){
     if(from.__x){ Alpine.clone(from.__x, to) }
   }
 }
})

Reply:
I made the change described above.